### PR TITLE
#89 [REQUEST] Add fields for facilitator and editor to the story archetype

### PR DESCRIPTION
**Pull request by LuvvAggarwal**
Solution Description:
Added "editor" property in stories.md archetype and created "credits.html" partial for templated with conditional rendering
Finally calling "credits" template in "single.html"
SS of final result
image

### CODE REVIEW

File: archetypes/stories.md
Comment (juliaferraioli) [inline archetypes/stories.md:1]: perfect, thanks!

File: content/stories/2023/thea-flowers.md
Comment (juliaferraioli) [inline content/stories/2023/thea-flowers.md:1]: This should be "Claire Moss", not "Clarie Moss".

File: layouts/partials/credits.html
Comment (juliaferraioli) [inline layouts/partials/credits.html:1]: Could we add a period at the end of the sentence?

File: layouts/stories/single.html
Comment (juliaferraioli) [inline layouts/stories/single.html:1]: Just for cleanliness of the code, could you only add in line 48?

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
Thanks so much! This is looking great. Added some comments to the files.

However, I think you accidentally committed more files thank you intended to. Mind removing the `.vs` folder from the PR?

File: layouts/stories/single.html
Comment (LuvvAggarwal) [reply layouts/stories/single.html:1]: Sure, would make the necessary changes

Comment (juliaferraioli) [reply layouts/stories/single.html:1]: I'm sorry, I think I communicated that poorly! Your approach of creating a new partial was great. Let's stick with that.

I was referencing the fact that it seems like you modified the indentation for lines that you didn't change, so the diff reflected changes on lines without material changes. No worries though, if you can just add back the partial, we can resolve formatting later.

File: .gitignore
Comment (juliaferraioli) [inline .gitignore:1]: Thanks so much for adding these!

**Review by juliaferraioli — CHANGES_REQUESTED** [review]

File: layouts/stories/single.html
Comment (LuvvAggarwal) [reply layouts/stories/single.html:1]: Created a new partial and calling it in single.html

**Review by juliaferraioli — APPROVED** [review]
Thanks! This looks great!

**Pull request merged**